### PR TITLE
Mcs 226 (Added receptacle points that scale with collider size)

### PR DESCRIPTION
**Pull request by jaudick**
Screenshot from 2020-06-22 16-01-06
Screenshot from 2020-06-22 16-00-03

https://nextcentury.atlassian.net/browse/MCS-226

### CODE REVIEW

File: unity/Assets/Scripts/Contains.cs [hunk 1]
```diff
@@ -246,18 +246,25 @@ public List<ReceptacleSpawnPoint> GetValidSpawnPoints(bool ReturnPointsCloseToAg
...
-        // TODO MCS-226 Adjust number of spawn points by receptacle trigger box size
 		//so lets make a grid, we can parametize the gridsize value later, for now we'll adjust it here
 		int gridsize = 4; //number of grid boxes we want, reduce this to SPEED THINGS UP but also GET WAY MORE INACCURATE
 		float xBoundsMin = triggerBoxCollider.bounds.min.x;
 		float xBoundsMax = triggerBoxCollider.bounds.max.x;
+		float zBoundsMin = triggerBoxCollider.bounds.min.z;
+		float zBoundsMax = triggerBoxCollider.bounds.max.z;
 
 		float xBoundsRange = Mathf.Abs(xBoundsMax - xBoundsMin);
+		float zBoundsRange = Mathf.Abs(xBoundsMax - xBoundsMin);
```
Comment (deanwetherby) [inline unity/Assets/Scripts/Contains.cs:257]: Should this be using the zBoundsMax and zBoundsMin?

Comment (jaudick) [reply unity/Assets/Scripts/Contains.cs:257]: Yes I will fix that

File: unity/Assets/Scripts/Contains.cs [hunk 2]
```diff
@@ -246,9 +246,31 @@ public List<ReceptacleSpawnPoint> GetValidSpawnPoints(bool ReturnPointsCloseToAg
...
+				gridsize += 1;
+				counterForGridSize += 0.1F;
+			}
+		} else {
+			while (counterForGridSize < zBoundsRange) {
+				gridsize += 1;
+				counterForGridSize += 0.1F;
+			}
+		}
+
```
Comment (chrislong) [inline unity/Assets/Scripts/Contains.cs:273]: I would use a temporary variable and assign it to max(xBoundsRange, zBoundsRange), and then the while loop doesn't need to be duplicated.

Or, can't this just be done with math? Something like `gridsize += (max(xboundsRange, zBoundsRange) - counterForGridSize) / 0.1`? (maybe with a `floor` or `ceil` in there)

Comment (jaudick) [reply unity/Assets/Scripts/Contains.cs:273]: You know, sometimes I just overthink the most basic solutions. This is a way better way of doing this, much simpler. Appreciate that you pointed it out.

**Pull request merged**